app/services/storage.js

```diff
@@ -373,6 +374,77 @@ export class ProfileStorage {
     return out;
   }
 
+  /**
+   * Search for places that match title, URL, or saved content
+   * since the specified timestamp.
+   *
+   * Snippets in the returned results are guaranteed to be HTML-escaped.
```

Comment:
And yes, the idea is that this API is generating HTML like it's 2007. The alternative is that we return some super awesome result format like:

```
[{t: "… some text containing "},
 {m: "a match"},
 {t: " then some more text."}]
```